tests/app/main/test_briefs.py

```diff
@@ -575,13 +575,22 @@ def test_create_new_brief_response(self, data_api_client):
         assert res.status_code == 302
         assert res.location == "http://localhost/suppliers/opportunities/1234/responses/result?result=success"
         data_api_client.create_brief_response.assert_called_once_with(
-            1234, 1234, processed_brief_submission, 'email@email.com')
+            1234,
+            1234,
+            processed_brief_submission,
+            'email@email.com',
+            page_questions=[
+                'respondToEmailAddress', 'essentialRequirements', 'niceToHaveRequirements', 'availability', 'dayRate'
+            ]
+        )
+        data_api_client.submit_brief_response.assert_called_once_with(
+            6, 'email@email.com')
 
     def test_create_new_brief_response_shows_result_page_for_not_all_essentials(self, data_api_client):
```

Comment:
This results in a redirect so it's a little confusing the test name indicates it should result in a page type.